### PR TITLE
#19 Setup app theme

### LINKED ISSUE #19: Setup app theme

### PR DESCRIPTION
**Pull request by aimeudoxie**
### What this PR do

This pull request sets up the theme of the app by adding font colors in both light and dark mode , font style and the  components (buttons and icons) to the app configurations 

### Task completed

- [x] colors: Primary, secondary, action, success, warning, danger and neutral
- [x] Font style : inter
- [x] components: buttons, icons
- [x] Theme toggle icon

### screen shots

Dark mode theme
IMG_7677

Light mode theme
IMG_7678

### CODE REVIEW

**Review by munezeromicha — APPROVED** [review]

**Review by fredshema — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -1,9 +1,115 @@
...
+        700:'var(--color-warning-700)',
+        600:'var(--color-warning-600)',
+        500:'var(--color-warning-500)',
+        400:'var(--color-warning-400)',
+        300:'var(--color-warning-300)',
+        200:'var(--color-warning-200)',
+        100:'var(--color-warning-100)',
+        50:'var(--color-warning-50)'
+      },
+      danger:{
```
Comment (ncutixavier) [inline tailwind.config.js]: Use error instead of danger

File: components/buttons.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Pressable, Text } from "react-native";
+
+interface Props {
+  title?: string;
+  size?: "S" | "L";
```
Comment (fredshema) [inline components/buttons.tsx]: Use conventional sizing, i.e: `sm, md, lg, xl`

File: components/buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  state?: "default" | "pressed" | "outlined" | "Hover" | "Disabled";
```
Comment (fredshema) [inline components/buttons.tsx]: Use consistent style, either use states that have the first letter as capitalized or make them all lowercase

**Review by fredshema — CHANGES_REQUESTED** [review]

**Review by fredshema — APPROVED** [review]

**Review by munezeromicha — APPROVED** [review]
Good job!

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**